modules/navier_stokes/doc/intro_slides/slides/index.md

```diff
@@ -0,0 +1,260 @@
+
+!col! width=50%
+
+Few Examples:
+
+- +PINSFV:+ +P+orous-Medium +I+ncompressible +N+avier-+S+tokes +F+inite +V+olume
+- +WCNSFV:+ +W+eakly-+C+ompressible +N+avier-+S+tokes +F+inite +V+olume
+
+!col-end!
+
+!row-end!
+
+!---
+
+# Building Input Files
+style=font-size:28pt
+
+The building blocks in MOOSE for terms in the PDEs are +Kernels+ for FE or +FVKernels+ for FV:
+
+!style! fontsize=80%
+!equation
+\underbrace{\frac{\partial \rho  \vec{u}}{\partial t}}_{\text{PINSFVMomentumTimeDerivative}}
++ \underbrace{\nabla \cdot \left(\gamma^{-1}  \rho \vec{u} \otimes \vec{u}\right)}_{\text{PINSFVMomentumAdvection}} =
+
+!equation
+\underbrace{\nabla \cdot \left(\gamma \mu_\text{eff} \left(\nabla\vec{u}_I + \left(\nabla \vec{u}_I\right)^T-\frac{2}{3}\nabla\cdot\vec{u}_I\mathbb{I}\right)\right)}_{\text{PINSFVMomentumDiffusion}}
+\underbrace{-\gamma\nabla p}_{\text{PINSFVMomentumPressure}}
++ \underbrace{\gamma\rho \vec{g}}_{\text{PINSFVMomentumGravity~}}
+\underbrace{-  W  \vec{u}_I}_{\text{PINSFVMomentumFriction}}
+
+The building blocks in MOOSE for boundary conditions are +BCs+ for FE or +FVBCs+ for FV:
+
+- Inlet Velocity: $\text{INSFVInletVelocityBC}$
+- Inlet Temperature: $\text{FVFunctionDirichletBC}$
+- Outlet pressure: $\text{INSFVOutletPressureBC}$
+- Heatflux: $\text{FVFunctionNeumannBC}$
+- Freeslip: $\text{INSFVNaturalFreeSlipBC}$
+- ...
+
+!style-end!
+
+!---
+
+# A Simple Example: Laminar Free Flow in a Channel
+style=font-size:26pt
+
+!media navier_stokes/nsfv-channel-example.png style=width:100%;background:white;
+
+Let us consider the following (fictional) material properties:
+
+- $\mu=1.1$ $\text{Pa}\cdot\text{s}$
+- $\rho=1.1$ $\frac{kg}{m^3}$ (we are using incompressible formulation in this case)
+
+!---
+
+# Detailed Input File
+style=font-size:26pt
+
+!listing modules/navier_stokes/test/tests/finite_volume/ins/channel-flow/2d-rc-no-slip.i
+
+!---
+
+# The Navier-Stokes Finite Volume Action
+style=font-size:26pt
+
+A simplified syntax has been developed, it relies on the `Action` system in MOOSE.
+For the documentation of the corresponding action, click [here](NSFVAction.md)!
+
+!listing modules/navier_stokes/test/tests/finite_volume/ins/channel-flow/2d-rc-no-slip-action.i
+
+!---
+
+# Recommendations for Building Input Files
+style=font-size:28pt
+
+!style! fontsize=80%
+
+- If possible, use the Navier Stokes Finite Volume Action syntax
+- Use Rhie-Chow interpolation for the advecting velocity
+
+  - Other interpolation techniques may lead to checker-boarding/instability
+
+
+- Start with first-order advected-interpolation schemes (e.g. upwind)
+- Make sure that the pressure is pinned for incompressible/weakly-compressible simulations in close-loop systems
+- For monolithic solvers (the default at the moment) use a variant of LU preconditioner
+- For complex monolithic systems monitor the residuals of every variable
+- Try to keep the number of elements relatively low (up to 1 million because LU preconditioner scales with $N^3$)
```

Comment:
```suggestion
- Try to keep the number of elements relatively low (up to 1 million because LU preconditioner scales with $N^3$). Larger problems can be solved in a reasonable wall-time as segregated-solve techniques are introduced
```